tests/phpunit/integration/tests/REST_API/Stories_Users_Controller.php

```diff
@@ -50,13 +58,28 @@ public static function wpSetUpBeforeClass( $factory ) {
 				'post_type'   => $post_type,
 			]
 		);
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		$this->controller = new \Google\Web_Stories\REST_API\Stories_Users_Controller();
+
+		$this->set_permalink_structure( '/%postname%/' );
```

Comment:
Do we still need this?

Reply:
This is done here https://github.com/google/web-stories-wp/blob/29a4bd1ef802e40059cfe16e8e77ff80ebde7854/tests/phpunit/integration/includes/Test_REST_TestCase.php#L53